go.mod

```diff
@@ -513,6 +513,8 @@ replace (
 //
 // replace github.com/dapr/components-contrib => ../components-contrib
 // replace github.com/dapr/kit => ../kit
+replace github.com/dapr/durabletask-go => github.com/cicoyle/durabletask-go v0.0.0-20250618220139-02008f613541
```

Comment:
[I'll rm this once the durabletask-go PR is merged](https://github.com/dapr/durabletask-go/pull/26)

Reply:
Ill fix conflicts and update this commit tomorrow